src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/ExecPlanGraph/Comparison/SkeletonNode.cs

```diff
@@ -80,5 +82,51 @@ public Graph GetGraph()
             return this.BaseNode.Graph;
         }
 
+        public SkeletonNodeDTO ConvertToDTO()
+        {
+            var skeletonNodeDTO = new SkeletonNodeDTO();
+            LevelOrderCopy(this, skeletonNodeDTO);
+
+            return skeletonNodeDTO;
+        }
+
+        private static void LevelOrderCopy(SkeletonNode skeletonNode, SkeletonNodeDTO skeletonNodeDTO)
```

Comment:
The name isn't really clear what it's doing. It seems to just be converting a SkeletonNode into a DTO? I'd just put this logic into the ConvertToDTO method - it doesn't really seem useful to have it split out like this.